packages/create-flex-plugin/src/lib/cli.ts

```diff
@@ -0,0 +1,75 @@
+import yargs, { Argv, Options } from 'yargs';
+import createFlexPlugin, { FlexPluginArguments } from './create-flex-plugin';
+
+const usage = [
+    'Creates a new Twilio Flex Plugin project',
+    '',
+    'Arguments:',
+    'name\tName of your plugin. Needs to start with plugin-'
+].join('\r\n');
+
+export interface CLIArguments {
+  accountSid?: string;
+  a?: string;
+  runtimeUrl?: string;
+  r?: string;
+  install?: boolean;
+  i?: boolean;
+  name?: string;
+  yarn?: boolean;
+  y?: boolean;
+}
+
+class cli {
+  private readonly parser: Argv<CLIArguments>;
+  private options: {[key: string]: Options;} = {
+    accountSid: {
+      alias: 'a',
+      describe: 'The Account SID for your Flex Project',
+      default: null,
+    },
+    runtimeUrl: {
+      alias: 'a',
+      type: 'boolean',
+      default: false,
+      describe: 'Auto-install dependencies'
+    },
+    install: {
+      alias: 'i',
+      type: 'boolean',
+      default: false,
+      describe: 'Auto-install dependencies'
+    },
+    yarn: {
+      alias: 'y',
+      type: 'boolean',
+      default: false,
+      describe: 'Use yarn as your dependency manager',
+    },
+    help: {
+      alias: 'h',
+      description: usage
+    },
+    version: {
+      alias: 'h'
+    }
+  };
+
+  constructor(cwd) {
```

Comment:
This is an implicit `any` so you might want to immediately clean up the house when moving to TS by turning strict on